openquake/qa_tests_data/event_based_risk/case_6c/expected/agg_curves_eb.csv

```diff
@@ -1,8 +1,10 @@
-#,,,,,,"generated_by='OpenQuake engine 3.9.0-gitc9a4f7bc4f', start_date='2020-03-05T07:08:25', checksum=2742411261, kind='agg_curves-rlzs', risk_investigation_time=1.0"
-return_period,rlz,loss_type,NAME_1,loss_value,loss_ratio,annual_frequency_of_exceedence
-50,0,structural,Region A,5.77686E+02,2.88843E-02,2.00000E-02
-50,0,structural,RegionB,5.74161E+02,1.14832E-02,2.00000E-02
-100,0,structural,Region A,7.73143E+02,3.86572E-02,1.00000E-02
-100,0,structural,RegionB,9.12244E+02,1.82449E-02,1.00000E-02
-200,0,structural,Region A,1.11514E+03,5.57572E-02,5.00000E-03
-200,0,structural,RegionB,2.85112E+03,5.70225E-02,5.00000E-03
+#,,,,,,"generated_by='OpenQuake engine 3.10.0-git94907f9121', start_date='2020-06-15T06:43:33', checksum=3986088084, kind='agg_curves-rlzs', risk_investigation_time=1.0"
+return_period,rlz,loss_type,site_id,loss_value,loss_ratio,annual_frequency_of_exceedence
+50,0,structural,1,1.38872E+03,1.38872E-01,2.00000E-02
+50,0,structural,2,1.26232E+02,INF,2.00000E-02
```

Comment:
Loss ratios for `site_id=2` are `INF`. The exposed value summation by `site_id` might be buggy.

Reply:
I know Anirudh, this is still work in progress.